src/opm/output/eclipse/AggregateUDQData.cpp

```diff
@@ -356,5 +474,46 @@ captureDeclaredUDQData(const Opm::Schedule&                 sched,
             iGph::staticContrib(igph[index], i_igph);
         }
         
+    std::size_t i_wudq = 0;
+    const auto& wnames = sched.wellNames(simStep);
+    const auto nwmax = nwmaxz(inteHead);
+    for (const auto& udq_input : udqCfg.input()) {
+        if (udq_input.var_type() ==  UDQVarType::WELL_VAR) {
+            const std::string& udq = udq_input.keyword();
+            auto i_dudw = this->dUDW_[i_wudq];
+            dUdw::staticContrib(st, wnames, udq, nwmax, i_dudw);
+            i_wudq++;
+        }
+    }
+    
+    // Make ordered list of current groups
+    std::vector<const Opm::Group2*> curGroups(ngmaxz(inteHead), nullptr);
```

Comment:
This construction is done twice - could extract it to a helper function? NB `Group2::insert_index()` has offset 0 - should probably *not* have the `-1`?

Reply:
Agree - I have made a helper function in the file instead